src/main/kotlin/org/usfirst/frc/team4099/robot/Robot.kt

```diff
@@ -94,6 +94,18 @@ class Robot : IterativeRobot() {
             val moveDown = controls.moveDown
             val toggle = controls.toggle
 
+            if (operator.moveDown && moveUp) {
+                operator.moveDown = false
+            } else if (!operator.moveDown && moveDown) {
+                operator.moveDown = true
+            }
+
+            elevator.updatePosition()
```

Comment:
this should be given an argument ex. updatePosition(wantsUp :  Boolean) so that the elevator actually knows where it wants to go; when you fix this replace moveUp and moveDown in elevator.updatePosition, because those variables don't exist